src/errors.js

```diff
@@ -113,3 +113,7 @@ export const CosmosBroadcastError = {
   "16": createCustomErrorClass(" CosmosBroadcastCodeGasOverflow"),
   "17": createCustomErrorClass(" CosmosBroadcastCodeNoSignatures"),
 };
+
+export const TransactionWasRejected = createCustomErrorClass(
+  "TransactionWasRejected"
```

Comment:
what do you mean by Reject? If it's user rejecting on the device, please reuse UserRefusedOnDevice

Reply:
My bad I didn't know that errors exists. Fixed !